Enforce singular hash params being sent in perform (#10)
1debit/simplekiq: README.md

Use `include Simplekiq::Worker` instead of `Sidekiq::Worker`

> You can _only_ pass _one_ hash to perform it should be named params, all parameters will have `symbolized_keys` so write tests and implementation accordingly.

Reviewer: ```suggestion
You can _only_ pass _one_ hash to the `perform` method. They should be named params, and `symbolized_keys` get's applied to the hash, so write tests and implementation accordingly.
```

> Workers should be _very_ simple, they should find or initaitialize another object and run _one_ method on it.

Reviewer: ```suggestion
Workers should be _very_ simple. They should find or initaitialize another object and run _one_ method on it.
```
